### PR TITLE
Math magicians: fetch data from API

### PR DESCRIPTION
**Pull request by ragangithub**
- Used the following external API: [API Ninjas](https://api-ninjas.com/api/quotes)
- Created a new component called Quote
- Fetched data inside of a new component
- Showed a loading state
- Showed an error state

### CODE REVIEW

File: src/components/Quote.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  if (hasError) return <div>Something went wrong!</div>;
+
+  if (isLoading) return <div>Loading...</div>;
+
+  return (
+  // <ul>
+  //   {data.slice(0, 3).map((item) => (
+  //     <li key={item.id}>{item.title}</li>
+  //   ))}
+  // </ul>
```
Comment (AlphaNtihinduka) [inline src/components/Quote.js]: - [ ] Please consider removing the codes commented. These make your codes look messy and too many comments can make code harder to read and understand 😞.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @ragangithub 🙋‍♂️
Good job working on math Magecian application. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- useEffect hook used correctly ✔️  
- Loading state implemented properly ✔️ 
- Well-structured files ✔️ 

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- Kindly consider the inline review comments ☺️.

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by Olamarx — APPROVED** [review]
Hi @ragangithub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- All changes effected ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ragangithub** [comment]
- Removed Inline comment

**Pull request merged**